### PR TITLE
Home page redesign

### PR DESCRIPTION
**Pull request by Typher7**
Carousel added to landing page.

### CODE REVIEW

File: src/routes/+page.svelte [hunk 1]
```diff
@@ -1,22 +1,38 @@
...
+		</Heading>
+	</div>
+
+	<div class="md:col-span-2 justify-between px-4 pb-10">
+		<div class="w-full lg:px-44">
+			<Carousel
+				class="flex h-[550px] justify-center items-center"
+				{images}
+				duration="7000"
+				let:Controls
```
Comment (Brogoming) [inline src/routes/+page.svelte:24]: Remove the indicators and controls, when scrolling the lay over the nav bar

File: src/routes/+page.svelte [hunk 2]
```diff
@@ -1,22 +1,38 @@
...
+
+		<Heading
+			tag="h2"
+			class="text-2xl font font-extrabold md:text-3xl lg:text:4xl text-slate-100 text-center"
+		>
+			Where ALL WSC Students are Welcome!
+		</Heading>
+	</div>
+
+	<div class="md:col-span-2 justify-between px-4 pb-10">
```
Comment (Brogoming) [inline src/routes/+page.svelte]: I would like there to be no padding on the x-axis and shorten the gap between the carousel and the words

File: src/routes/+page.svelte [hunk 3]
```diff
@@ -1,22 +1,38 @@
...
+	import images from '../routes/images.json';
 </script>
 
-<p class="text-center text-2xl pb-8">
-	WSC ACM Student Chapter offers the opportunity to meet peers and experts in many fields of
-	interest. With a world-wide membership, ACM is a leading resource for computing professional and
-	students working in various fields of information technology.
-</p>
-<Gallery items={images} class="grid gap-4 grid-cols-3" />
+<div class="grid md:grid-cols-3 gap-8 px-4">
```
Comment (Brogoming) [inline src/routes/+page.svelte]: I would like there to be no padding on the x-axis, I would like the components to be flush with the sides

File: src/routes/+page.svelte [hunk 4]
```diff
@@ -1,22 +1,38 @@
...
+				{images}
+				duration="7000"
+				let:Controls
+				let:Indicators
+			>
+				<Controls class="px-10"/>
+				<Indicators />
+			</Carousel>
+		</div>
+		<p class="text-center text-xl py-20">
```
Comment (Brogoming) [inline src/routes/+page.svelte]: py-20 seems a bit much

**Review by Brogoming — CHANGES_REQUESTED** [review]
Delete duplicate photos, make these changes then we will go from there, I'm not 100% sure about our initial idea but do these changes and we can go from there

File: src/routes/+page.svelte (hunk 1 above)
Comment (Typher7) [reply src/routes/+page.svelte:24]: Do you mean hiding them on hover?

File: src/routes/+page.svelte (hunk 4 above)
Comment (Typher7) [reply src/routes/+page.svelte]: halved it to 10

File: src/routes/+page.svelte (hunk 3 above)
Comment (Typher7) [reply src/routes/+page.svelte]: removed it, but the 'y-axis' padding comes from the layout

File: src/routes/+page.svelte (hunk 1 above)
Comment (Brogoming) [reply src/routes/+page.svelte:24]: no full removal because scrolling they layover the nav bar

File: src/routes/+page.svelte (hunk 3 above)
Comment (Brogoming) [reply src/routes/+page.svelte]: We can work on that

File: src/routes/+page.svelte (hunk 1 above)
Comment (Typher7) [reply src/routes/+page.svelte:24]: I've fixed the layover problem, let me know if you still want the ctrls and ind removed

**Review by ACMGitHub — APPROVED** [review]

**Pull request merged**